From 7902951fa08395c372417f76158439f0c689fa28 Mon Sep 17 00:00:00 2001
From: Bo Shen <voice.shen@atmel.com>
Date: Fri, 30 May 2014 14:21:26 +0800
Subject: [PATCH] obp-utils: remove -Werror flags

Signed-off-by: Bo Shen <voice.shen@atmel.com>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 567fcee..3ec7a65 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -13,7 +13,7 @@ AM_CFLAGS = \
 	-fvisibility=hidden \
 	-ffunction-sections \
 	-fdata-sections \
-  -Wall -Werror -Wmissing-declarations -Wmissing-prototypes -Wnested-externs \
+  -Wall -Wmissing-declarations -Wmissing-prototypes -Wnested-externs \
   -Wpointer-arith -Wsign-compare -Wchar-subscripts -Wstrict-prototypes \
   -Wwrite-strings -Wshadow -Wformat-security -Wtype-limits \
   -DMXT_VERSION=\"$(GIT_VERSION)\" \
-- 
1.8.5.2

